sys-boot/gnu-efi/gnu-efi-3.0.6-r1.ebuild

```diff
@@ -29,6 +29,18 @@ RDEPEND=""
 QA_EXECSTACK="usr/*/lib*efi.a:* usr/*/crt*.o"
 RESTRICT="strip"
 
+pkg_pretend() {
+	if use custom-cflags; then
+		ewarn
+		ewarn "You have enabled building with USE=custom-cflags. Be aware that"
```

Comment:
Sure. I actually think that is way better. I would recommended doing the same for the rEFInd ebuild as well. I copied that msg from there, by the way, with slight modifications. :) If you don't mind, I would adapt both and update the pull request appropriately tomorrow.